### ISSUE TITLE
Failed to load workflow - /spec 400

### ISSUE THREAD

**Issue by MattNewberry** [bitrise-io/bitrise-workflow-editor#38]
`GET http://localhost:3645/api/spec`

#### Request Headers
```
GET /api/spec HTTP/1.1
Host: localhost:3645
Connection: keep-alive
Pragma: no-cache
Cache-Control: no-cache
Accept: application/json, text/plain, */*
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36
DNT: 1
Referer: http://localhost:3645/
Accept-Encoding: gzip, deflate, sdch, br
Accept-Language: en-US,en;q=0.8
```

#### Response Headers
```
HTTP/1.1 400 Bad Request
Content-Type: application/json
Date: Fri, 03 Feb 2017 16:22:22 GMT
Content-Length: 32
```

#### Response Body
```json
{"error":"Missing output data"}
```

**Comment by erosdome** [comment]
Hi, thanks for reporting the issue, it is related to the release process. A new version vill arrive shortly:

https://discuss.bitrise.io/t/offline-workflow-editor-workflow-editor-v2-v0-9-1-released/626

**Comment by godrei** [comment]
Hi @MattNewberry ,
sorry for the faulty release. 
We shared a new version with fixed plugin definition: 0.9.2

**Comment by viktorbenei** [comment]
Let us know if you'd still have any this issue with v0.9.2 @MattNewberry , and thanks for reporting the issue! ;)
